packages/sliver-client.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Sliver-Client'
+$category = 'Command & Control'
```

Comment:
looks like there's an issue with the category lint

Reply:
https://github.com/mandiant/VM-Packages/blob/main/scripts/test/lint.py#L306-L340

https://github.com/mandiant/VM-Packages/blob/main/categories.txt

Reply:
Hi, has anyone run into this issue with the linter? Copied straight from the categories.txt for the selected category.